Fibonacci.java

```diff
@@ -0,0 +1,18 @@
+public class Fibonacci {
+
+	public static void main(String[] args) { 
+		int nthFib = Integer.parseInt(args[0]);
+		System.out.println(new Fibonacci().findNthFib(nthFib));
+	}
+
+	private int findNthFib(int nthFib) {
+		int currentFib = 1;
+		int prevFib = 0;
```

Comment:
change currentFib = 0 and prevFib = 1, first two number in all fib sequences are 1 0 ...